module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -3487,4 +3557,103 @@ protected function checkTitleHoldRules(array $bib, array $patron): bool
         }
         return false;
     }
+
+    /**
+     * Gets title information for holds placed in an INN-Reach system
+     *
+     * @param $holdId the id of the hold from Sierra
+     * @param $bibId  the id of the bib from Sierra
+     *
+     * @return array
+     *
+     * @throws ILSException
+     */
+    protected function getInnReachHoldTitleInfoFromId($holdId, $bibId): array
+    {
+
+        $this->getInnReachDb();
```

Comment:
Assign the return value to a variable and use it here. Like:

```
$db = $this->getInnReachDb();
...
if ($db) ...

```

...and remove the extra blank line above. :)